WooCommerce/WooCommerceTests/ViewRelated/Orders/Order Details/Customer Note/EditCustomerNoteViewModelTests.swift

```diff
@@ -30,18 +30,32 @@ class EditCustomerNoteViewModelTests: XCTestCase {
         assertEqual(viewModel.navigationTrailingItem, .done(enabled: true))
```

Comment:
I have updated the current test cases and created new ones, which cover when the feature flag is on or off.